packages/backend/patch/orbitdb-oplog.patch

```diff
@@ -0,0 +1,15 @@
+--- packages/backend/node_modules/@orbitdb/core/src/oplog/log.js	2024-10-03 11:17:39
```

Comment:
This resolves an issue where entries that are generated by a peer aren't saved locally and when that peer is offline it throws a ton of errors and crashes.